src/main/java/seedu/sudohr/logic/commands/DeleteCommand.java

```diff
@@ -26,6 +26,10 @@ public class DeleteCommand extends Command {
 
     private final Index targetIndex;
 
+    /**
+     * Creates an DeleteCommand to delete employee at the specified
+     * {@code targetIndex}
+     */
     public DeleteCommand(Index targetIndex) {
```

Comment:
Can you change the argument to Event Title? Should be applicable to EditEvent, AddEmployeetoEvent, DeleteEmployeeFromEvent, ListEmployeeInEvent as well. Thanks!